files/grest/apispec/koiosapi.yaml

```diff
@@ -2038,7 +2058,6 @@ components:
             example: 23
     asset_summary:
       type: array
-      description: An array of assets contained on input UTxO
```

Comment:
Description deleted?

Reply:
Yes, cause it was copied from another endpoint. It doesn't fit:
![image](https://user-images.githubusercontent.com/12885604/153214224-ed026f75-e178-468b-8e29-2d2098a1218f.png)

I wasn't sure what to replace it with so I just removed it (it would be the subtitle under schema in the image)

Reply:
> cause it was copied from another endpoint

gotcha 👍🏻